src/main/java/study/step2/Expression.java

```diff
@@ -0,0 +1,32 @@
+package study.step2;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+public class Expression {
+
+    private final List<Integer> numbers;
+    private final List<Operator> operators;
+
+    public Expression(List<Integer> numbers, List<Operator> operators) {
+        this.numbers = numbers;
+        this.operators = operators;
+    }
+
+    public void addNumber(int number) {
+        numbers.add(number);
+    }
+
+    public void addOperator(Operator operator) {
+        operators.add(operator);
+    }
+
+    public List<Integer> getNumbers() {
+        return new ArrayList<>(numbers);
```

Comment:
```suggestion
        return Collections.unmodifiableList(numbers);
```
수정, 삭제로부터 안전한 불변객체를 반환하는건 어떨까요

Reply:
아 이런것도 있었네요 👍 
참고하겠습니다!